schema/chapter.json

```diff
@@ -24,46 +24,49 @@
       "type": "string"
     },
     "events": {
-      "description": "Event URLs related to the chapter.",
-      "type": "array",
+      "description": "Events related to the project.",
       "items": {
-        "format": "uri",
-        "type": "string"
+        "$ref": "common.json#/definitions/event"
       },
       "minItems": 1,
+      "type": "array",
       "uniqueItems": true
     },
     "leaders": {
       "description": "Leaders of the chapter.",
-      "type": "array",
       "items": {
         "$ref": "common.json#/definitions/person"
       },
       "minItems": 2,
+      "type": "array",
       "uniqueItems": true
     },
     "logo": {
       "additionalProperties": false,
       "description": "Logo for the chapter.",
-      "type": "array",
       "items": {
         "$ref": "common.json#/definitions/logo"
       },
       "minItems": 1,
+      "type": "array",
       "uniqueItems": true
     },
     "mailing_list": {
```

Comment:
The naming is not following the convention here -- if it's an array it should be plural.